tests/kubectl_test.go

```diff
@@ -145,6 +146,47 @@ var _ = Describe("[sig-compute]oc/kubectl integration", func() {
 			Entry("[test_id:3466]virtualmachineinstance", "get", "vmi", "wide", []string{"NAME", "AGE", "PHASE", "IP", "NODENAME", "READY", "LIVE-MIGRATABLE", "PAUSED"}, 1, "True"),
 		)
 
+		It("should verify vmi ip value match primary interface ip value", func() {
+			Expect(createBridgeNetworkAttachmentDefinition(virtCli)).ToNot(HaveOccurred())
+
+			vm1 := tests.NewRandomVirtualMachine(tests.NewRandomVMI(), false)
+			vm1, err = virtCli.VirtualMachine(util.NamespaceTestDefault).Create(vm1)
+			Expect(err).NotTo(HaveOccurred())
+			tests.StartVirtualMachine(vm1)
```

Comment:
Can you please fix `StartVirtualMachine` in a commit so it will return the updates VMI? (so we will avoid the next two lines)

Reply:
This PR is back to draft, I'm fixing some issues so this code is temporary. I'll let you know when finished